packages/frontend/src/components/accounts/ledger/SignInLedgerWrapper.js

```diff
@@ -40,12 +39,10 @@ export function SignInLedgerWrapper(props) {
 
     const [accountId, setAccountId] = useState('');
     const [loader, setLoader] = useState(false);
-    const [path, setPath] = useState(1);
-    const [confirmedPath, setConfirmedPath] = useState(null);
-    const ledgerHdPath = confirmedPath ? `44'/397'/0'/0'/${confirmedPath}'` : null;
+    const [confirmedPath, setConfirmedPath] = useState(1);
+    const ledgerHdPath = `44'/397'/0'/0'/${confirmedPath}'`;
```

Comment:
naive question: but is this alright to be hard coded? @marcinbodnar

Reply:
It should probably be some constant (I will make this change), but yes, we are using this path for Wallet, and `44'/397'/0'/0'/` prefix is always the same.